app/helpers/podcasts_helper.rb

```diff
@@ -5,6 +5,18 @@ module PodcastsHelper
 
   RSS_LANGUAGE_CODES = %w[af sq eu be bg ca zh-cn zh-tw hr cs da nl nl-be nl-nl en en-au en-bz en-ca en-ie en-jm en-nz en-ph en-za en-tt en-gb en-us en-zw et fo fi fr fr-be fr-ca fr-fr fr-lu fr-mc fr-ch gl gd de de-at de-de de-li de-lu de-ch el haw hu is in ga it it-it it-ch ja ko mk no pl pt pt-br pt-pt ro ro-mo ro-ro ru ru-mo ru-ru sr sk sl es es-ar es-bo es-cl es-co es-cr es-do es-ec es-sv es-gt es-hn es-mx es-ni es-pa es-py es-pe es-pr es-es es-uy es-ve sv sv-fi sv-se tr uk]
 
+  def feed_description(feed, podcast)
+    [feed.description, podcast.description].detect { |d| d.present? } || ""
+  end
+
+  def episode_description(episode)
+    if episode.podcast.has_apple_feed?
+      episode.description_safe
+    else
+      episode.description_with_default
+    end
+  end
```

Comment:
That look better @svevang ?

If there is an apple feed associated with the podcast
    publishes the "safe" description (truncated with fallbacks),
otherwise
    uses the description (with fallbacks, no truncation)